### PR TITLE
REF: ToolTipMenu to TSX

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/qf6q6dvrx2ehwrc2wus4h33c3i?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1656)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/iitwxql2gypyd4t4vjzmbu7rfi?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1657)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/hroyvdbnkemth66alta2y5ve74?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1658)

File: components/QRCodeComponent.tsx [hunk 1]
```diff
@@ -68,30 +64,13 @@ const QRCodeComponent: React.FC<QRCodeComponentProps> = ({
   };
 
   const onPressMenuItem = (id: string) => {
-    if (id === actionKeys.Share) {
+    if (id === 'share') {
```
Comment (Overtorment) [inline components/QRCodeComponent.tsx:67]: looks like it got worse. used to besome constant (type safe), now just arbitrary string

File: components/types.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { ViewStyle } from 'react-native';
+
+export interface Action {
+  id: string | number;
```
Comment (Overtorment) [inline components/types.ts:4]: maybe pure string is better?

**Review by Overtorment — APPROVED** [review]

File: components/types.ts (hunk 2 above)
Comment (marcosrdz) [reply components/types.ts:4]: looks like it but..I just decided to simply put number to avoid TSLint

File: components/QRCodeComponent.tsx (hunk 1 above)
Comment (marcosrdz) [reply components/QRCodeComponent.tsx]: addressed

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/upqvioid6ru2jf5m2nzm4ufiue?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1659)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/tjwb7awlgc3bhpszdoznqalc2y?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1662)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/cgsuvyclp7se6yhpvrmsvxnx5q?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1667)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/6fvf5mw2flln6ualdyv6746dlm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1668)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**